lib/travis/build/script/scala.rb

```diff
@@ -6,8 +6,8 @@ class Script
       class Scala < Jvm
 
         DEFAULTS = {
-          scala: '2.10.4',
-          jdk:   'default'
+          scala: '2.12.2',
+          jdk:   'oraclejdk8'
```

Comment:
Sorry, I did not know about this requirement.

In this case, then, we will postpone merging this until we can move to Java 8 for the default JVM version.